pytorch_blade/tests/torch-disc-pdll/tests/fake_quant.pdll

```diff
@@ -0,0 +1,63 @@
+/// Pre-defined custom call prototypes
+///
+/// const std::string kDefaultHelperFunctionDeclarations = R"pdll(
+///   Rewrite PackValue_1(tag : Attr, v0 : Value) -> ValueRange;
+///   Rewrite PackValue_2(tag : Attr, v0 : Value, v1 : Value) -> ValueRange;
+///   Rewrite UnpackValue_1(v : ValueRange) -> (Value);
+///   Rewrite UnpackValue_2(v : ValueRange) -> (Value, Value);
+///   Rewrite CreateTorchCustomCall(tag : Attr, inputs : ValueRange, outputs : ValueRange) -> (op: Op, new_outputs : ValueRange);
+///   Rewrite SetAttr(op : Op, key : Attr, value : Attr);
+///   Rewrite SetCustomAttr(op : Op, key : Attr, value : Attr);
+/// )pdll";
+
+// A new Pattern
+Pattern TorchFakeQuantOp {
+  /// match phase: define the pattern
+  let fake_quant_op = op<torch.operator>(
+    input: Value,
+    scale: Value,
+    zero_point: Value,
+    quant_min: Value,
+    quant_max: Value,
+    num_bits: Value,
+    axis: Value,
+    signed: Value,
+    symmetric: Value,
+    dynamic: Value,
+    per_channel: Value
+  ){ name = attr<"\"torch_blade.fake_quant\"">} -> (old_type: Type);
+
+  /// rewrite phase
+  rewrite fake_quant_op with {
+    let new_type = ConvertTorchTensorElemType(old_type, attr<"\"i8\"">);
```

Comment:
using i8 or ui8 depends on `signed` attribute. We also needs to `num_bits` into consideration as well.

Reply:
Agree, it is a problem deserves considering to make quantize-related pattern match more robust.